3rdparty/carotene/CMakeLists.txt

```diff
@@ -20,8 +20,12 @@ if(CMAKE_COMPILER_IS_GNUCC)
     # - matchTemplate about 5-10%
     # - goodFeaturesToTrack 10-20%
     # - cornerHarris 30% for some cases
-
-    set_source_files_properties(${carotene_sources} COMPILE_FLAGS "--param ipcp-unit-growth=100000 --param inline-unit-growth=100000 --param large-stack-frame-growth=5000")
+    set_source_files_properties(${carotene_sources} COMPILE_FLAGS "--param inline-unit-growth=100000 --param large-stack-frame-growth=5000")
+    if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "10.0.0")
+	    set_source_files_properties(${carotene_sources} COMPILE_FLAGS "--param ipcp-unit-growth=100000 ${COMPILE_FLAGS}")
+    else()
+	    set_source_files_properties(${carotene_sources} COMPILE_FLAGS "--param ipa-cp-unit-growth=100000 ${COMPILE_FLAGS}")
```

Comment:
Please use single call of `set_source_files_properties(${carotene_sources} COMPILE_FLAGS ...)`

```
if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "10.0.0")
  set(carotene_sources_flags "--param ipcp-unit-growth=100000")
else()
  set(carotene_sources_flags "--param ipa-cp-unit-growth=100000")
endif()
set_source_files_properties(${carotene_sources} COMPILE_FLAGS "--param inline-unit-growth=100000 --param large-stack-frame-growth=5000 ${carotene_sources_flags}")
```

Reply:
done